spec/test_doubles.rb

```diff
@@ -102,6 +102,32 @@ def self.do_something_with_no_starting_context
     end
   end
 
+  class NestedOrganizer
+    extend LightService::Organizer
+
+    def self.call(context)
+      context[:foo] = [1, 2, 3]
```

Comment:
Here is my main underlying question: is an Organizer supposed to basically also be an action, or is it supposed to apply a bunch of actions on, and then return, a context?